### DISCUSSION TITLE
实验报告名单没名字,但是之前有交,怎么办?

### DISCUSSION THREAD

**Discussion by daga-1024** [zhangyuanes/OperatingSystemExperiments#15] [Q&A]
第四次的实验报告交给学委了,但是名单没看到名字,不知道还来不来得及补交.故此打扰助教老师一下.

**Comment by zhangyuanes** [comment]
> 第四次的实验报告交给学委了,但是名单没看到名字,不知道还来不来得及补交.故此打扰助教老师一下.

可以提交，请提供当时提交的记录，证明是按时提交的，但是由于部分问题导致未准时，我们会按照正常提交批改。如果没有按照补交批改。
